custom_components/waste_collection_schedule/waste_collection_schedule/source/eastleigh_gov_uk.py

```diff
@@ -23,18 +23,35 @@
 }
 
 
-API_URL = "https://eastleigh.gov.uk/waste-bins-and-recycling/collection-dates/your-waste-bin-and-recycling-collections"
+API_URL = "https://www.eastleigh.gov.uk/waste-bins-and-recycling/collection-dates/your-waste-bin-and-recycling-collections"
 
 
 class Source:
     def __init__(self, uprn: str | int):
         self._uprn: str | int = uprn
 
     def fetch(self):
+        s = requests.Session()
+        s.headers.update({
+            "User-Agent": "Mozilla/5.0 (Linux; Android 6.0; Nexus 5 Build/MRA58N) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/141.0.0.0 Mobile Safari/537.36",
+            "Accept": "text/html,application/xhtml+xml,application/xml;q=0.9,image/avif,image/webp,image/apng,*/*;q=0.8,application/signed-exchange;v=b3;q=0.7",
+            "Accept-Encoding": "gzip, deflate, br, zstd",
```

Comment:
I'd prefer to just remove `br` here
it works for me, and we don't need additional dependencies